playwright/async_api/_generated.py

```diff
@@ -9472,6 +9472,36 @@ async def launch_persistent_context(
             )
         )
 
+    async def connect(
+        self, abc: str, *, timeout: float = None, slow_mo: float = None
```

Comment:
abc ? should be `ws_endpoint`

Reply:
This was just temporarily, with the next generation (next push9 its ws_endpoint.